.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
             - storybook-cache--master--{{ checksum ".circleci/config.yml" }}
       - run:
           name: Run Chromatic
-          command: npm run chromatic
+          command: npm run chromatic -- --exit-zero-on-changes --auto-accept-changes=master
```

Comment:
Q: What does `--exit-zero-on-changes` do? The docs make it sound like this will switch chromatic visual changes from being a pending check to being a failing check, is that correct?